Send warning when no Python tests are discovered

It can be inconvenient to have a `FATAL_ERROR` raised when no tests are discovered:
```console
> cmake --build ./build
[ 50%] Building CXX object src/CMakeFiles/example.dir/main.cpp.o
[100%] Linking CXX shared module example.so
[100%] Built target example
CMake Error at /path/to/pytest-cmake/cmake/PytestAddTests.cmake:93 (message):
  No Python tests have been discovered.
```

A warning would be sufficient.

--- a/cmake/PytestAddTests.cmake
+++ b/cmake/PytestAddTests.cmake
@@ -92,7 +92,7 @@ if(CMAKE_SCRIPT_MODE_FILE)
         endforeach()
 
         if(NOT _content)
-            message(FATAL_ERROR "No Python tests have been discovered.")
+            message(WARNING "No Python tests have been discovered.")
         endif()
     endif()
 

